src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -57,11 +58,11 @@ const ReportActionItemSingle = (props) => {
         login,
         pendingFields,
     } = props.personalDetails[props.action.actorEmail] || {};
-    const avatarUrl = props.action.automatic
+    const avatarSource = props.action.automatic
         ? CONST.CONCIERGE_ICON_URL
 
         // Use avatar in personalDetails if we have one then fallback to avatar provided by the action
-        : (avatar || props.action.avatar);
+        : (ReportUtils.getCorrectAvatar(avatar, props.action.actorEmail));
```

Comment:
Seems fishy we're not looking at `props.action.avatar` anymore here, no?

Reply:
It points to the default old dot avatar, so we don't need to use it here

Reply:
Noice 👍